### PR TITLE
Renaming

### PR DESCRIPTION
**Pull request by daxpedda**
Some of the `navigation::*` can be skipped and imported by `use`, but I left that to somebody who knows the code much better then I do.

### CODE REVIEW

File: shared/src/protocol/navigation/mod.rs [hunk 1]
```diff
@@ -1,86 +1,5 @@
...
-
-pub use self::{flight_plan::*, piloting::*};
-use crate::solar_systems::{Solar, SolarSystemId};
-
 mod flight_plan;
+mod location;
 mod piloting;
 
-#[derive(Debug, Copy, Hash, Clone, Eq, PartialEq, Serialize, Deserialize)]
-pub struct SolarSystemLocationId(pub i64);
```
Comment (ecton) [inline shared/src/protocol/navigation/mod.rs:14]: This type was lost, or it became named something wrong. I think it became SolarSystemId, which really sounds the same as SystemId -- I think it needs to have either Object or Location in the name.

File: web/src/app.rs [hunk 2]
```diff
@@ -90,7 +87,7 @@ pub enum Message {
     ToggleRendering,
     ForegroundGame,
     LogOut,
-    NavigateToLocation(SolarSystemId, SolarSystemLocationId),
+    NavigateToLocation(SystemId, navigation::SolarSystemId),
```
Comment (ecton) [inline web/src/app.rs]: This is the usage where the other comment made me realize something went wrong with renaming. The first parameter is an id pointing to the solar system itself, the second is the id of the object within the solar system that you're navigating to.

File: shared/src/protocol/navigation/mod.rs (hunk 1 above)
Comment (daxpedda) [reply shared/src/protocol/navigation/mod.rs:14]: This is one of the `SolarSystem` to `System` that I apparently forgot to rename back again.

File: web/src/app.rs (hunk 2 above)
Comment (daxpedda) [reply web/src/app.rs]: :+1:

File: shared/src/protocol/navigation/mod.rs (hunk 1 above)
Comment (daxpedda) [reply shared/src/protocol/navigation/mod.rs:14]: Fixed.

**Pull request merged**